auth-web/src/store/modules/org.ts

```diff
@@ -923,7 +923,8 @@ export default class OrgModule extends VuexModule {
   @Action({ rawError: true })
   public async resetAccountWhileSwitchingPremium (): Promise<void> {
     this.context.commit('setGrantAccess', false)
-    this.context.commit('setCurrentOrganization', { name: '' })
+    // need to keep accesstype while resetting to know whenther the account is GOVM or not.
+    this.context.commit('setCurrentOrganization', { ...this.context.state['currentOrganization'], ...{ name: '' } })
```

Comment:
while resetting before we are setting name as null, no we need accesstype in currentorg. If needed we can remove resetting all and add only access type from state.
any thoughts?